src/ml_tooling/transformers.py

```diff
@@ -246,3 +247,39 @@ def fit(self, X: pd.DataFrame, y=None):
     def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         data = self.scaler.transform(X)
         return pd.DataFrame(data=data, columns=X.columns, index=X.index)
+
+
+class DFRowFunc(BaseEstimator, TransformerMixin):
+    """
+    Row-wise operation on Pandas DataFrame. Strategy can either be one of
+    the predefined or a callable.    If some elements in the row are NaN these
+    elements are ignored for the built-in strategies.
+    """
+
+    _func_map = {'sum': np.sum,
+                 'min': np.min,
+                 'max': np.max}
+
+    def __init__(self, strategy=None):
+
+        if strategy is None:
+            raise TransformerError("No strategy is specified.")
+
+        if not isinstance(strategy, str) and not callable(strategy):
+            raise TransformerError(f"{strategy} is not a callable or a string.")
+
+        if isinstance(strategy, str) and strategy not in self._func_map.keys():
```

Comment:
One more nitpick 😄 
```suggestion
        if isinstance(strategy, str) and strategy not in self._func_map:
```

The check for "in" defaults to dictionary keys